mitodl/smoot-design
# fix: export module augmentation for consuming applications

### Description (What does it do?)
Enables smoot-design's module augmentation to work in consuming applications.

### How can this be tested?
1. Run `yarn install` and `yarn storybook`, read http://localhost:6006/?path=/docs/smoot-design-themeprovider--docs
2. Clone https://github.com/mitodl/unified-ecommerce-frontend, checkout branch `cc/setup`, and observe that in `Header.tsx`, the `theme.custom` props come through.
